### PR TITLE
Fixed Grammatical Errors and Typos

### PR DESCRIPTION
**Pull request by LoftyDroid**

### CODE REVIEW

**Comment by MrKrishnaAgarwal** [comment]
Will get back to you soon!

**Comment by LoftyDroid** [comment]
Hey @MrKrishnaAgarwal can you please accept my PR.

**Comment by MrKrishnaAgarwal** [comment]
Why did you remove the stash section @LoftyDroid?

File: README.md [hunk 1]
```diff
@@ -1,312 +1,273 @@
...
-
-Stash Changes with a message
-`git stash save "message"`
-<br>
-List all stashes
-`git stash list`
-<br>
-Apply the most recent stash and remove it from the stash list.
-`git stash pop`
+See ```git help git``` for an overview of the system.
```
[the remark this answers is not in the document]
Comment (MrKrishnaAgarwal) [reply README.md]: Resolved!

**Review by MrKrishnaAgarwal — APPROVED** [review]
I've resolved everything!

Thanks a lot for contributing!!

**Pull request merged**